src/betterproto/plugin.py

```diff
@@ -1,480 +0,0 @@
-#!/usr/bin/env python
```

Comment:
split up into `src/betterproto/plugin/__init__.py` and `src/betterproto/plugin/parser.py`